docs/job_defines_format.md

```diff
@@ -0,0 +1,43 @@
+# Jenkins Job Definitions
+
+Due to a growing need of having extra metadata available to jenkins consumers
+(both users and automation) a job define system was added to SCT jenkins generator.
+
+It replaces previous `_display_name` file (which used to contain just the display
+ name of the folder) with a special .yaml file called `_jobDefines.yaml`.
+
+Example definitions file:
+
+```yaml
+# used by argus to determine argus plugin
+job-type: scylla-cluster-tests
+# used by argus to determine which parameter wizard to use on execution
+job-sub-type: longevity
+# replacement for _display_name file, sets the jenkins folder display name
+folder-name: Cluster - Longevity Tests
+# Per job (regex supported) job overrides, defines as a mapping
+overrides:
```

Comment:
I think `overrides`, should be limited to `job-sub-type` 

I don't want people to be lazy and use it for `job-description`

Reply:
Hmm, what's wrong with using it for job description as well? How else they could add a text description to a specific job?

Reply:
job description should come from the jenkinsfile themselves, we shouldn't put all the description in the same file, too much housekeep, for operation like moving a job from one folder to another

Reply:
I agree with @fruch here.
BTW, somehow related , I had a feature request somewhere to add like metadata tags to the test yamls, so Argus could present them and also search test cases according to them.

Reply:
Done